src/_pytest/main.py

```diff
@@ -391,22 +412,30 @@ class Session(nodes.FSCollector):
     _setupstate = None  # type: SetupState
     # Set on the session by fixtures.pytest_sessionstart.
     _fixturemanager = None  # type: FixtureManager
+    exitstatus = None  # type: Union[int, ExitCode]
```

Comment:
Didn't find any place which does `hasattr('exitstatus')` or such, so I think this is safe.

Reply:
It's still kind of an API change.
But I also think it is good to have it being set always.